client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -124,24 +110,9 @@ class TaxonomyManagerListItem extends Component {
 					}
 				</span>
 				{ ! isUndefined( term.post_count ) && <Count count={ term.post_count } /> }
-				<span
-					className="taxonomy-manager__action-wrapper"
-					onClick={ this.togglePopoverMenu }
-					ref="popoverMenuButton">
-					<Gridicon
-						icon="ellipsis"
-						className={ classNames( {
-							'taxonomy-manager__list-item-toggle': true,
-							'is-active': this.state.popoverMenuOpen
-						} ) } />
-				</span>
-				<PopoverMenu
-					isVisible={ this.state.popoverMenuOpen }
-					onClose={ this.togglePopoverMenu }
-					position={ 'bottom left' }
-					context={ this.refs && this.refs.popoverMenuButton }
-				>
-					<PopoverMenuItem onClick={ this.editItem } icon="pencil">
+				<EllipsisMenu position={ 'bottom left' }>
```

Comment:
Any particular reason to use an attribute expression vs. just `position="bottom left"` string value?